### PR TITLE
Fixed the links which navigate to 404

### PR DESCRIPTION
**Pull request by RobertChang0722**
Hi,

I just found some links already outdated on the README, and fixed it.

But I can't find the `C` repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ The project provides two interfaces: the libSQL API, which supports all the feat
 
 To get started with the libSQL API:
 
-* [JavaScript](https://github.com/libsql/libsql-experimental-node)
-* [Rust](core) 
+* [JavaScript](https://github.com/libsql/libsql-js)
+* [Rust](https://github.com/libsql/libsql-client-rs)
```
Comment (penberg) [inline README.md:38]: Hey! The Rust repository you're pointing to is actually something different and you should keep the existing link.

File: README.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ The project provides two interfaces: the libSQL API, which supports all the feat
...
 To get started with the libSQL API:
 
-* [JavaScript](https://github.com/libsql/libsql-experimental-node)
-* [Rust](core) 
+* [JavaScript](https://github.com/libsql/libsql-js)
+* [Rust](https://github.com/libsql/libsql-client-rs) 
 * [Python](https://github.com/libsql/libsql-experimental-python) (experimental)
-* [Go](bindings/go) (experimental)
+* [Go](https://github.com/libsql/libsql-client-go) (experimental)
 * [C](bindings/c) (experimantal)
```
Comment (penberg) [inline README.md:41]: FYI this is the correct link to the C source code.

**Review by penberg — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (khinshankhan) [reply README.md:41]: is it perhaps `crates/bindings/c` <https://github.com/tursodatabase/libsql/tree/main/crates/bindings/c>?

**Comment by penberg** [comment]
The pull request has conflicts and has not been updated for a while. Please feel free to open a new one if you wish to submit this for merging.

**Pull request closed without merging**